tests/e2e/pom/home-page-model.ts

```diff
@@ -20,11 +20,26 @@ export class DevHomePage {
     }
 
     /**
-     * Opens the account settings page.
+     * Opens the user settings page.
      */
-    async openAccountSettings() {
-        await expect(this.page.getByRole("heading", { name: "Settings" })).not.toBeVisible();
+    async openSettings() {
         await this.page.getByRole("button", { name: /[A-Z]{2}/ }).click();
         await this.page.getByRole("link", { name: "Settings" }).click();
     }
+
+    /**
+     * Opens the account settings page.
+     */
+    async openAccountSettings() {
+        this.openSettings();
+        await this.page.getByRole("link", { name: "Account" }).click();
+    }
+
+    /**
+     * Opens the shortcuts settings page.
+     */
+    async openShortcutsSettings() {
+        this.openSettings();
```

Comment:
You still need to await this
```suggestion
        await this.openSettings();
        await this.page.getByRole("link", { name: "Account" }).click();
    }

    /**
     * Opens the shortcuts settings page.
     */
    async openShortcutsSettings() {
        await this.openSettings();
```

Reply:
You're absolutely right.